src/main/java/com/akadk/repository/GroupRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.akadk.repository;
+
+import com.akadk.entity.Group;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface GroupRepository extends JpaRepository<Group, Long> {
+    Group findById(long id);
```

Comment:
FindById already implemented at JpaRepository with plenty others methods. So you don't have to mention them here. Also related to others repo.